src/test/java/org/opensearch/security/ssl/OpenSSLTest.java

```diff
@@ -22,7 +22,6 @@
 import java.util.Set;
```

Comment:
These tests were widely untested, waiting for the latest test run to confirm if this updated behaviors works correctly in linux jdk11 runs

This is one of the last items to remove the 'Draft' state of this pull request

Reply:
Resolved!